services/server/src/config/local-test.js

```diff
@@ -15,7 +15,10 @@ module.exports = {
   },
   storage: {
     read: RWStorageIdentifiers.SourcifyDatabase,
-    writeOrWarn: [RWStorageIdentifiers.RepositoryV1],
+    writeOrWarn: [
+      RWStorageIdentifiers.RepositoryV1,
+      WStorageIdentifiers.S3Repository,
```

Comment:
Can we enable this also in staging already?

Reply:
What do you mean? This config is just for tests, and they are passing